### PR TITLE
#2433 Remove unused dependencies from the worker image

### PR DESCRIPTION
**Pull request by vitojeng**
required by #2433 

### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #issue_number)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-it/build.gradle [hunk 1]
```diff
@@ -48,12 +49,16 @@ dependencies {
     exclude group: 'com.sun.jersey', module: 'jersey-json'
     exclude group: 'com.sun.jersey', module: 'jersey-servlet'
     exclude group: 'com.sun.jersey', module: 'jersey-server'
+    exclude group: 'org.mortbay.jetty', module: 'jetty'
+    exclude group: 'org.mortbay.jetty', module: 'jetty-util'
   }
```
Comment (vitojeng) [inline ohara-it/build.gradle]: Should add:
```
  exclude group: 'org.apache.zookeeper', module: 'zookeeper'
```

File: ohara-kafka/build.gradle [hunk 2]
```diff
@@ -34,6 +34,8 @@ dependencies {
     exclude group: 'com.sun.jersey', module: 'jersey-json'
     exclude group: 'com.sun.jersey', module: 'jersey-servlet'
     exclude group: 'com.sun.jersey', module: 'jersey-server'
+    exclude group: 'org.mortbay.jetty', module: 'jetty'
+    exclude group: 'org.mortbay.jetty', module: 'jetty-util'
   }
```
Comment (vitojeng) [inline ohara-kafka/build.gradle]: Should add:
```
  exclude group: 'org.apache.zookeeper', module: 'zookeeper'
```

File: ohara-testing-util/build.gradle [hunk 3]
```diff
@@ -46,12 +47,16 @@ dependencies {
     exclude group: 'com.sun.jersey', module: 'jersey-json'
     exclude group: 'com.sun.jersey', module: 'jersey-servlet'
     exclude group: 'com.sun.jersey', module: 'jersey-server'
+    exclude group: 'org.mortbay.jetty', module: 'jetty'
+    exclude group: 'org.mortbay.jetty', module: 'jetty-util'
   }
```
Comment (vitojeng) [inline ohara-testing-util/build.gradle]: Should add:
```
  exclude group: 'org.apache.zookeeper', module: 'zookeeper'
```

**Comment by chia7712** [comment]
the error is caused by #226, and it will be resolved by #2459

**Comment by vitojeng** [comment]
@chia7712 Could you take a look again?

**Comment by chia7712** [comment]
> Could you take a look again?

let me retry failed module :)

**Comment by chia7712** [comment]
retry assembly

**Comment by vitojeng** [comment]
> let me retry failed module :)

Ok, jsut let me know when you done.

**Pull request merged**